site/content/docs/main/deploy-options.md

```diff
@@ -7,16 +7,29 @@ If you don't have a cluster with that capability see the [Running without a Kube
 
 ## Installation
```

Comment:
I would love to see this page/section augmented further - right now it goes over conceptual stuff around daemonset vs deployment, service types, etc., but it doesn't actually walk through installing Contour from the manifests, via the Operator, via Helm, etc., and how you could use each of those to get the desired configuration of Contour/Envoy. Probably can be a separate issue, it just reads strangely to me by skipping over the nuts and bolts of installing and jumping from conceptual to "test your installation".